### ISSUE TITLE
Transcript Modal > Window Options button

### ISSUE THREAD

**Issue by kb-smarterbalanced** [ableplayer/ableplayer#499]
Hello, we have used AblePlayer on our site: smartertoolsforteachers.org and we had an accessibility review done. This is one of the accessibility modifications that were suggested.

**Transcript Modal > Window Options button**
This is updating correctly for mouse users, but when triggered by keyboard or screen reader, it does not update correctly. 
Ensure that the `aria-expanded` attribute correctly reflects the state of the expandable content. The value of the attribute should be set to `"true"` when the content is expanded and `"false"` when the content is collapsed.

**Comment by terrill** [comment]
Issue 1:  The `aria-expanded` attribute is now being toggled as expected. This is fixed in 4b7b56bf8ee2a8bac903fd0a8f459d2844838636

There were some bugs in how its state was being determined. Also worth noting though: Prior to this fix, when the pop-up was closed, `aria-expanded` was removed altogether, rather than being set to false. This was once the W3C-recommended practice, although it was [confusing and controversial](https://github.com/w3c/aria-practices/issues/1318). From my testing, it has the same effect on screen readers as setting `aria-expanded="false"`, but to avoid further confusion I've changed this practice throughout Able Player, and now the attribute is always present where applicable, and is always either 'true' or 'false'.

**Comment by terrill** [comment]
**Issue 3:** I disagree with this. The "Window Options" menu is a menu, in that its options (Move, Resize, and Close) provide functionality; they aren't web links. The link you provided to an article on navigation menus requires a password.  Based on your link text and URL though, I think it's probably discussing why not to use role="menu" for a web navigation menu, which I agree with.  In fact, I too have written about this in [Dropdown Menus 2020: Menus are not menus](https://terrillthompson.com/1226).  In this case though, these aren't web navigation links, they're functions, and I'm confident that `role="menu"` (and related ARIA) is appropriate.

**Comment by terrill** [comment]
**Issue 4:** I disagree with this (sorry if I sound like I'm being disagreeable! I appreciate all this feedback, and it's forcing me to think twice about the choices we've made).  The items in this menu have `role="menuitem"`. Therefore, they aren't buttons; they're menu items (different semantics).  See the [Actions Menu Button](https://www.w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-actions.html) example from the APG.

**Comment by terrill** [comment]
**Issue 5:** - I created a new issue for making the "Move" interface more accessible. It's #537

**Comment by terrill** [comment]
**Issues 6 and 7:** I created a new issue for these, #538

**Comment by terrill** [comment]
@kb-smarterbalanced that should fix all these issues. Those that aren't fixed have each been moved to a new issue.  I'm closing this now, but feel free to reopen, or create new individual issues, if you spot additional problems. Thanks again for all the feedback.

**Comment by kb-smarterbalanced** [comment]
Thank you so much for all your work on this!
